console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/util/Constants.java

```diff
@@ -22,4 +22,6 @@ private Constants() {
     public static final int LABEL_WIDTH_USER_FORM = 445;
     public static final int LABEL_WIDTH_INFO = 150;
     public static final int LABEL_WIDTH_FORM_EXPLANATION = 600;
+    public static final int LABEL_WIDTH_UPLOAD_AND_APPLY = 75;
```

Comment:
Is this correct naming for constants as others are starting with type (LABLE for example) then metric (WIDTH) then context (USER) and then type again (FORM).
What is UPLADE_AND_APPLY?